src/main/java/net/mcreator/element/parts/gui/SizedComponent.java

```diff
@@ -26,11 +26,13 @@
 public abstract class SizedComponent extends GUIComponent {
 
 	public int width, height;
+	public boolean changesHeight;

```

Comment:
This parameter should not be serialized as this is a property of component type, not component type instance.

Even better solution would be to make this a method, canChangeHeight as this way this problem is avoided altogether. Make it return false by default and override in your component

Reply:
Call it "canChangeHeight" to match javanic naming set/get/is/has/can